woocommerce/woocommerce
# Minify updated PayPal admin JS.

### Changes proposed in this Pull Request:

Re-minifies `/gateways/paypal/assets/js/paypal-admin.js`. This file was [updated here](https://github.com/woocommerce/woocommerce/commit/387e095a01284401484f1810447aa38cf246ab2c) to address issues with deprecated jQuery functionality, but because this asset lives in a somewhat non-standard path it was missed by the build script.

I did not add it to the build path, because this gateway is in essential maintenance mode only and the recommendation is to switch to a different solution (though, it still seemed worth addressing since there are definitely users out there who still rely on it).

Closes #31244.

### How to test the changes in this Pull Request:

1. Try these steps with `SCRIPT_DEBUG` alternately set to `false` and `true`.
2. Visit **WooCommerce ▸ Settings ▸ Payments ▸ PayPal Standard**.
3. Toggle **Enable PayPal sandbox** on and off.
4. You should find that the **API credentials** section (further down on the same page) updates accordingly, exposing fields for either the live or sandbox credentials as appropriate.

https://user-images.githubusercontent.com/3594411/158444885-b0658e62-fd20-4286-9589-62d345b570db.mp4

### Other information:




### Changelog entry

> Tweak - Update PayPal Standard JS used in the admin environment to avoid deprecated functionality.

### FOR PR REVIEWER ONLY:
